Configuration/Settings.yaml

```diff
@@ -1,5 +1,6 @@
 Flowpack:
   ElasticSearch:
+    version: 2.x
```

Comment:
Would the default version 1.x break most installations? Not sure if we should set 1.x or 2.x to default?! For sure this option is nowhere else used atm but could be in the future

Reply:
@patricweihmayer You run Elasticsearch 5.4.1? With which version of the CR adaptor? 😯